### ISSUE TITLE
fix NPE when call GET stats for credentials that have not been used yet

### ISSUE THREAD

**Issue by bambulin** [whalebone/microfactor#6]
```
microfactor_1       | May 25, 2020 2:39:50 PM io.whalebone.microfactor.rest.ErrorHandler unexpectedError
microfactor_1       | SEVERE: Unexpected error caught
microfactor_1       | java.lang.NullPointerException: temporal
microfactor_1       | 	at java.base/java.util.Objects.requireNonNull(Objects.java:246)
microfactor_1       | 	at java.base/java.time.LocalDate.from(LocalDate.java:392)
microfactor_1       | 	at java.base/java.time.LocalDateTime.from(LocalDateTime.java:456)
microfactor_1       | 	at io.whalebone.microfactor.utils.TimestampUtils.isTheSameMinute(TimestampUtils.java:26)
microfactor_1       | 	at io.whalebone.microfactor.service.MicroFactorService.getCredentialsStats(MicroFactorService.java:77)
microfactor_1       | 	at io.whalebone.microfactor.service.$MicroFactorServiceDefinition$Intercepted.$$access0(Unknown Source)
```

**Comment by rattuscz** [comment]
@bambulin quite urgent now for testing :-)

**Comment by bambulin** [comment]
fixed by d2d4382
